packages/distribution/addon/abilities/inquiry.js

```diff
@@ -4,101 +4,129 @@ import { DateTime } from "luxon";
 
 import config from "@projectcaluma/ember-distribution/config";
 
+/**
+ * This class contains all permission definitions for inquiries. To improve
+ * performance there are a few helpers and rules for optimal permission
+ * computation. The permissions need to be ordered by how expensive their
+ * computation is: the least expensive first and the most expensive last and so
+ * on:
+ *
+ * 1. Static config properties (e.g. `enableReminders`)
+ * 2. Base permission which checks for the correct task and the readonly config
+ * 3. Simple work item property checks (e.g. `isReady` or `isSuspended`)
+ * 4. Addressed / controlling group affiliation (e.g. `isAddressed` or `isControlling`)
+ * 5. All other computations (e.g. whether the deadline is overdue)
+ * 6. Custom permissions served by the host app (using `hasCustomPermission`)
+ */
 export default class InquiryAbility extends Ability {
   @service calumaOptions;
 
   @config config;
 
-  get canEdit() {
+  hasCustomPermission(permissionName, ...args) {
+    return this.config.permissions[permissionName]?.(...args) ?? true;
+  }
+
+  get hasBasePermission() {
     return (
       !this.config.ui.readonly &&
-      this.model?.task.slug === this.config.inquiry.task &&
-      ["SUSPENDED", "READY"].includes(this.model?.status) &&
-      this.model?.controllingGroups
-        .map(String)
-        .includes(String(this.calumaOptions.currentGroupId))
+      this.model?.task.slug === this.config.inquiry.task
+    );
+  }
+
+  get isReady() {
+    return this.model.status === "READY";
+  }
+
+  get isSuspended() {
+    return this.model.status === "SUSPENDED";
+  }
+
+  get isAddressed() {
+    return this.model.addressedGroups
+      .map(String)
+      .includes(String(this.calumaOptions.currentGroupId));
+  }
+
+  get isControlling() {
+    return this.model.controllingGroups
+      .map(String)
+      .includes(String(this.calumaOptions.currentGroupId));
+  }
+
+  get canEdit() {
+    return (
+      this.hasBasePermission &&
+      // Since editing in the status ready has the same character as sending an
+      // inquiry, we need to make sure that permission would be given
+      (this.isSuspended || (this.isReady && this.canSend)) &&
```

Comment:
this.canSend will only be true if the inquiry is suspended, no?

Reply:
fixed